auto_round/auto_quantizer.py

```diff
@@ -303,7 +307,7 @@ def convert_model(self, model: nn.Module):
         quantization_config = model.config.quantization_config
         bits = quantization_config.bits
         group_size = quantization_config.group_size
-        data_type = quantization_config.data_type
+        data_type = quantization_config.data_type if hasattr(quantization_config, "data_type") else "int"
```

Comment:
why changing here

Reply:
some gptq models like qwen2-1.5b-int4-inc does not have this attribute.